scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -141,6 +141,9 @@ SG_ Steering_Angle : 0|32@1+ (1,0) [-110|110] "deg" DCM
 BO_ 312 FSM_BRAKE_PRESSURE_SENSOR: 4 FSM
 SG_ Brake_Pressure: 0|32@1+ (1,0) [0|2500] "psi" DEBUG 
 
+BO_ 313 FSM_BRAKE_OC_SC: 1 FSM
+SG_ Brake_Is_OC_SC: 0|1@1+ (1,0) [0|1] "" DEBUG 
```

Comment:
I would probably put this in `BO_ 312` instead of making a new message

Reply:
Okay this should be good